### DISCUSSION TITLE
The old app had a badge when friends were online. Would others prefer to have this in the new app?

### DISCUSSION THREAD

**Discussion by ijm8710** [lichess-org/mobile#814] [Ideas]
I found this badge very useful in the old app and had requested it last year. Another user also recently requested it. Veloce did say "no" but I had two objectives with this post

1) Veloce came around on the game matrix due to a lot of other users missing it. If this gets traction perhaps he can here as well😉
2) if there is a specific reason as to why he opposes it I fully respect that and would love to hear why. Seems Veloce has in the past provided some view of his thoughts in discussion posts and when we asked on discord it was only "no" so was hoping to hear more about this if possible

**Comment by EmmetSchuler** [comment]
If by badge you mean a number / indicator by the friends icon, I would definitely like to see that in the app

**Comment by ijm8710** [comment]
Exactly, like this user mocked up ![image](https://github.com/lichess-org/mobile/assets/25938701/567327fd-0f53-4234-bedc-1b43b0838a2f)

**Comment by veloce** [comment]
I don't know if we can do that because it adds more strain to server load. This is the reason why the website doesn't show friends by default too.

**Reply by ijm8710** [reply]
That's a fair enough answer. My only follow up would it make a difference if it only showed for the existence of 1+? Meaning if at least one friend was online, badge it....rather than fetching a true actual realtime count?

**Reply by veloce** [reply]
We can't do that without creating the connection to the server so it doesn't change anything.
